### ISSUE TITLE
A11y_SQL Azure Data Studio_New Terminal_Keyboard:Tab key or shift+tab does not move to link "https://aka.ms/pscore6" is n 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#12984]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**: #A11yMAS;#A11yTCS;#A11ySev1;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.1.1;#Benchmark;#Keyboard;#AINotInScope;#AILimited;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to view menu item in the menu bar and use down key arrow to select item "terminal"
4. Use shift+tab to navigate to  https://aka.ms/pscore6

**Actual:**
On navigating using shift+tab or tab to link https://aka.ms/pscore6 keyboard key tab cannot access it.

**Expected:**
Tab and shift+tab from the first line of command should move to link https://aka.ms/pscore6

**User Impact:**
Keyboard users will not be able to access the link.

**Attachment:**
[#12984_A11y_SQL Azure Data Studio_New Terminal_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/5401669/12984_A11y_SQL.Azure.Data.Studio_New.Terminal_Keyboard.zip)

**Comment by alanrenmsft** [comment]
first of all, the feature mentioned in the issue is a command line interface, I don't think command line interface works like a normal UI based application, so closing as by design.

**Comment by fsteffi** [comment]
@alanrenmsft  the command line as well need to be accessible the link in it cant be accessed so keyboard users will not be able to click on it and use it

please reopen this issue

#Regressed:02-20-23;

**Comment by alanrenmsft** [comment]
@fsteffi as per our discussion, these are really old bugs, please reopen so that we can reset the SLA status.

**Comment by Roopesh-T** [comment]
logged new bug https://github.com/microsoft/azuredatastudio/issues/22333.

** GitHub Tags**: #closed;
